src/metabase/pulse/render/table.clj

```diff
@@ -74,14 +75,21 @@
   [score]
   (int (* (/ score 100.0) max-bar-width)))
 
+(def ^:private max-column-character-length 100)
+
+(defn- truncate-text [text]
+  (if (> (count text) max-column-character-length)
+    (str (str/trim (subs text 0 max-column-character-length)) "...")
```

Comment:
Ahh, @adam-james-v thanks for this! I didn't take care of the trailing spaces, but this does it 👍